src/Header/Header.jsx

```diff
@@ -1,22 +1,20 @@
 import { NavLink } from "react-router-dom";
+import { useAuth } from "../services/auth";
 import "./Header.scss";
 
 function Header() {
+    const { user } = useAuth();
```

Comment:
because we specified `value={{ user, setUser }}` in AuthContext.Provider and wrapped the entire app in AuthContext.Provider, we can access the context values from anywhere